rust/account/src/account.rs

```diff
@@ -17,9 +17,11 @@
 use snarkvm_wasm::{
     account::{Address as AleoAddress, PrivateKey as AleoPrivateKey, ViewKey as AleoViewKey},
     network::Testnet3,
+    program::{Ciphertext as AleoCiphertext, Record as AleoRecord},
 };
 
 pub type CurrentNetwork = Testnet3;
 pub type Address = AleoAddress<CurrentNetwork>;
 pub type PrivateKey = AleoPrivateKey<CurrentNetwork>;
 pub type ViewKey = AleoViewKey<CurrentNetwork>;
+pub type Record = AleoRecord<CurrentNetwork, AleoCiphertext<CurrentNetwork>>;
```

Comment:
Maybe naming this type `RecordCiphertext` would avoid possible misunderstandings.